pytest_jira.py

```diff
@@ -162,6 +162,8 @@ def __init__(
             self.basic_auth = None
 
     def _jira_request(self, url, method='get', **kwargs):
+        if 'verify' not in kwargs:
+            kwargs['verify'] = self.verify
```

Comment:
Cool, I wasn't aware that requests supports this natively or I would have added it myself